src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -482,15 +458,13 @@ public void shellActivated(ShellEvent e)
         itemQuit.setText("Quit");
         itemQuit.addListener(SWT.Selection, new Listener()
         {
-            @Override
             public void handleEvent(Event e)
             {
                 shell.dispose();
             }
         });
         trayItem.addListener(SWT.MenuDetect, new Listener()
         {
-            @Override
             public void handleEvent(Event arg0)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)